lib/couchdb/index.js

```diff
@@ -84,23 +85,56 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findOneEqById = R.curry((couch, bucket, predicates) =>
+  getById(couch, bucket, predicates.id)
+  .then(R.when(Util.notNil, (doc) => R.compose(
+    R.ifElse(
+      R.equals(ID.couchify(predicates))
+    , R.always(doc)
+    , R.always(null)
+    )
+  , R.pick(R.keys(ID.couchify(predicates)))
```

Comment:
As soon as you have more than one statement in an es6 => {} block you need an explicit return ... And I noticed that we do the couchify twice, vs. other places where we only do the operation once based on the branch that is executed.

In those cases I sometimes suggest a variable for code readability, but again not as inefficient as doing something twice ... might be a non-issue, but something to keep in mind.